service/vspo-schedule/web/src/data/mocks/clips.ts

```diff
@@ -537,9 +540,9 @@ export const mockClips = [
     commentCount: "6",
     createdAt: "2023-03-20T09:00:00.000Z",
   },
-];
+].map((clip) => ({ ...clip, platform: clip.platform as Platform }));
```

Comment:
Maybe we can also consider to use [string literal types](https://www.typescriptlang.org/docs/handbook/literal-types.html#string-literal-types) to prevent mapping?

I don't have any preference though